pkg/image/minio.go

```diff
@@ -0,0 +1,51 @@
+package image
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// MinioImage looks through the nodes in the cluster and finds nodes that have already pulled Minio, and then finds the latest image tag listed
+func MinioImage(clientset kubernetes.Interface) (string, error) {
+	nodes, err := clientset.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return "", fmt.Errorf("failed to list nodes with minio image: %w", err)
+	}
+
+	bestMinioImage := ""
+	for _, node := range nodes.Items {
```

Comment:
What happens if image is in a private registry and has not been pulled yet?

Reply:
we changed the logic. Will send the PR out shortly.